src/features/ui/FormDetails/FormDetails.tsx

```diff
@@ -0,0 +1,29 @@
+import { FC } from 'react';
```

Comment:
FYI there's a `Wizard.NFTDetails` component coming shortly that will be able to replace this.

Reply:
Oh nice! haha I commented above saying a details wizard component would be handy 😁